### PR TITLE
[teleop] Auton GUI Updates

### PR DESCRIPTION
**Pull request by bli76**
- Add Clear Waypoints button
- Fix auton mode turning off when nav state is set to done
- Remove radio repeater
- Code cleanup
- Allow for teleop control when not in autonomy mode
- Log data with button

### CODE REVIEW

**Review by CameronTressler — APPROVED** [review]

File: base_station/gui/src/components/AutonTask.vue [hunk 1]
```diff
@@ -61,7 +59,7 @@ import OdometryReading from './OdometryReading.vue'
 import ArmControls from './ArmControls.vue'
 import DriveControls from './DriveControls.vue'
 import EncoderCounts from './EncoderCounts.vue'
-import WaypointEditor from './WaypointEditor_Auton.vue'
+import WaypointEditor from './WaypointEditor.vue'
 // import AutonJoystickReading from './AutonJoystickReading.vue'
```
Comment (zwgold) [inline base_station/gui/src/components/AutonTask.vue:63]: is this in use at all? If not, could we remove it

Comment (CameronTressler) [reply base_station/gui/src/components/AutonTask.vue]: Fixed

**Review by cgiger00 — APPROVED** [review]
can you guys upload screenshots next time of how the gui looks? hard to tell from just the code

**Comment by CameronTressler** [comment]
image

**Comment by CameronTressler** [comment]
> image

Flashes green when done. Can still enable auton mode when Done.

Currently working on getting drive code working (that's why you can see the speed delimiter instead of zed gimbal angle in the lower right. Another commit coming soon

**Comment by CameronTressler** [comment]
image

See the speed delimiter in the bottom left

**Comment by CameronTressler** [comment]
image

Button in lower left will save a csv file with odometry data. Planning to add logging for the rest of the data later today but this way you all can see the structure of the changes

**Comment by jnnanni** [comment]
image
New icon on a portion of the map with the background API

**Comment by cgiger00** [comment]
lgtm, lets wait for matt to review

**Comment by mmartin4972** [comment]
lgtm

**Pull request merged**